pkg/pyinst/bitmask.spec

```diff
@@ -20,19 +35,12 @@ exe = EXE(pyz,
           debug=False,
           strip=False,
           upx=True,
-          console=False )
+          console=False,
+          icon='../../data/images/mask-icon.ico')
 coll = COLLECT(exe,
-               a.binaries,
-               a.zipfiles,
-               a.datas,
-               strip=False,
-               upx=True,
-               name='bitmask')
-if sys.platform.startswith("darwin"):
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

agreed, but pyinstaller fails with the following message if the
condition on sys.platform is in the spec file:

```
Traceback (most recent call last):
  File "C:\Python27\Scripts\pyinstaller-script.py", line 9, in <module>
    load_entry_point('PyInstaller==3.1.1', 'console_scripts',
'pyinstaller')()
  File "C:\python27\lib\site-packages\PyInstaller\__main__.py", line
90, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "C:\python27\lib\site-packages\PyInstaller\__main__.py", line
46, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs
)
  File
"C:\python27\lib\site-packages\PyInstaller\building\build_main.py",
line 755, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'),
kw.get('clean_build'))
  File
"C:\python27\lib\site-packages\PyInstaller\building\build_main.py",
line 701, in build
    exec(text, spec_namespace)
  File "<string>", line 47, in <module>
NameError: name 'sys' is not defined
```

On 02/12/2016 03:42 PM, Kali Kaneko wrote:
> In pkg/pyinst/bitmask.spec 
> <https://github.com/leapcode/bitmask_client/pull/965#discussion_r52748
544>:
>
> 
>> coll = COLLECT(exe, -               a.binaries, -
>> a.zipfiles, -               a.datas, -
>> strip=False, -               upx=True, -
>> name='bitmask') -if sys.platform.startswith("darwin"):
> 
> this shouldn't be removed, if we're going to use the same file for
> both win and osx builds
> 
> — Reply to this email directly or view it on GitHub 
> <https://github.com/leapcode/bitmask_client/pull/965/files#r52748544>.
>
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQIcBAEBCAAGBQJWxbKgAAoJENwE0+YLGC5y8AEP/RUubP7ufuLDBkvXZO1jdHpE
d15m8QJULeGQPU+SPRuCFrXqCtTmudh8sLOdBe34JIJPDG19e/2PHW5psUpCktxh
AJuTA3F1Bm7aWpCESYKbniSmqjcgQjzF/XdaLaoXmRTJHyfWMCPxyybEn305OOZS
9YjC10wjfuP+B4lpXrQ6tXRMs2mAG8hDcmhyLNVzUlC7s+/KQPIpAe70u9B9dnvl
8eNNlYkWCbP/VlfUU82Y7YwvExwh7f3gs8NhIdLs+f5eRCraVL/JxyB7cQtNUbtp
uBL1zLtxUIBVa4p1lR9J2N+T6YWHI49JoTr0SGOMSpbOM8i+P11obmd2ey5GwPJF
mfjkT35bc0oUA61i8Hj9mFTBB0BtoDLqfavGgPb//fidjdYVKGHfQKFztWlQ/B8O
VAtK7vlglvDAz8On26ffW9LoqdsN49e66mOPJWhIwT9xBORV2/SB/fszvVE9ebDC
UUoCaWiX1FN2udqIA7sW2R3ADVrvCAMRKTGN/RtN7V1ibe3+/oZqNoW18X19zlnZ
mJh+SwXtl7GG6fNfzoKuiPmkz/89gQFZEmUwKXHIW6b99x9xnk2RmjpK1yvq9zfO
9AdGrxCuI9Cv9Yxj7Aw2k4pN7hgdkPPS2P+r4i/wdyoJfn1XwxxLqvsRZB/Yz9dP
XBXhRB5n6z1b0ObEQV7J
=ak1S
-----END PGP SIGNATURE-----